sensor/kubernetes/localscanner/tls_issuer_test.go

```diff
@@ -131,7 +131,7 @@ func TestLocalScannerTLSIssuerRefresherFailureStartFailure(t *testing.T) {
 
 	startErr := fixture.tlsIssuer.Start()
 
-	assert.NoError(t, startErr)
+	assert.Error(t, startErr)
```

Comment:
Typically more assertions are not necessary if an error occurred every following assertion fails.
```suggestion
	require.Error(t, startErr)
```

Reply:
Thanks for advise, fixed.